### PR TITLE
fix: adapt comment to corresponding used value

### PR DESCRIPTION
**Pull request by martinconic**
### Checklist

- [x] I have read the [coding guide](https://github.com/ethersphere/bee/blob/master/CODING.md).
- [ ] My change requires a documentation update, and I have done it.
- [ ] I have added tests to cover my changes.
- [x] I have filled out the description and linked the related issues.

### Description
Fixes [4823](https://github.com/ethersphere/bee/issues/4823)

### CODE REVIEW

**Review by janos — APPROVED** [review]

File: pkg/pullsync/pullsync.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ const (
 	pageTimeout                     = time.Second
 	makeOfferTimeout                = 15 * time.Minute
 	handleMaxChunksPerSecond        = 250
-	handleRequestsLimitRate         = time.Second / handleMaxChunksPerSecond // handle max 100 chunks per second per peer
+	handleRequestsLimitRate         = time.Second / handleMaxChunksPerSecond // handle max 250 chunks per second per peer
```
Comment (ldeffenb) [inline pkg/pullsync/pullsync.go:54]: Personally, I would have avoided redundancy with something like:
`handle MaxChunksPerSecond per second per peer`

**Review by ldeffenb — CHANGES_REQUESTED** [review]
Why mention the 250 twice?

File: pkg/pullsync/pullsync.go (hunk 1 above)
Comment (martinconic) [reply pkg/pullsync/pullsync.go]: good point

**Review by acha-bill — APPROVED** [review]

File: pkg/pullsync/pullsync.go [hunk 2]
```diff
@@ -51,7 +51,7 @@ const (
 	pageTimeout                     = time.Second
 	makeOfferTimeout                = 15 * time.Minute
 	handleMaxChunksPerSecond        = 250
-	handleRequestsLimitRate         = time.Second / handleMaxChunksPerSecond // handle max 100 chunks per second per peer
+	handleRequestsLimitRate         = time.Second / handleMaxChunksPerSecond // handle max `handleMaxChunksPerSecond` chunks per second per peer
```
Comment (ldeffenb) [inline pkg/pullsync/pullsync.go:54]: I like that much better as it clarifies that the rate is per second per peer (the latter being important) while not being redundant about the actual value.

**Review by ldeffenb — COMMENTED** [review]
Let 'er rip!

**Pull request merged**